### ISSUE TITLE
Manganato and mangakakalot expexted url scheme http or https

### ISSUE THREAD

**Issue by TheBeardOfTruth** [keiyoushi/tachiyomi-extensions-source#4623]
### Source information

Manganato 1.4.7

### Source language

English

### Steps to reproduce

Try to read or download a chapter of any series, it will fail to load.

### Expected behavior

Download should not error out.

### Actual behavior

![Screenshot_20240814-150044](https://github.com/user-attachments/assets/9c533c6f-be4b-408b-96bb-8d457836a09f)

### Mihon/Tachiyomi version

TachiyomiJ2K 1.7.4

### Android version

Android 14

### Other details

This also affects mangakakalot 1.4.8

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by AwkwardPeak7** [comment]
try changing server in webview and clear chapter cache

**Comment by TheBeardOfTruth** [comment]
It worked but only after doing it twice and restarting the software, as a workaround it's decent I suppose but it's still a strange bug (and I hit the same issue in webview when navigating the page)

**Comment by AwkwardPeak7** [comment]
Its a source issue, since you experienced it in webview as well
